app/src/main/res/values/strings.xml

```diff
@@ -810,16 +810,16 @@
     <string name="security">Security</string>
     <string name="set_mode_for_app_ops_dots">Set mode for app ops…</string>
     <string name="backup_source_description">Back up the entire source directory.</string>
-    <string name="backup_apk_only_description">Back up only the APK files from the source directory. <b>Source</b> must be selected for this to work.</string>
-    <string name="backup_data_description">Back up data folders. If selected, internal data folders will be backed up by default.</string>
-    <string name="backup_external_data_description">Back up external data folders. <b>Data</b> must be selected for this to work.</string>
+    <string name="backup_apk_only_description">Only backs up APK files from the source directory. (Select <b>Source</b> first.)</string>
+    <string name="backup_data_description">Backs up internal data folders by default.</string>
```

Comment:
This sentence can be interpreted in many ways which may lead some people to believe that the internal data folders are backed up regardless of the status of the flag. This option shouldn't use third person either.

Reply:
I don't understand how it could be misinterpreted. Certainly it could be misread.

Reply:
> I take it to mean "Turn this on → internal data folders are always backed up"

This what _you_ take. Others might interpret it differently.